platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/signed/StateToDiskReason.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.state.signed;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.Objects;
+
+/**
+ * The reason for writing the state to disk
+ */
+public enum StateToDiskReason {
```

Comment:
I would be ok if we just used the enum string, as opposed to providing a lower case version. Nobody currently depends on these exact strings (I think).

Reply:
To clarify, you mean it's ok to use the enum strings for the directories where ISS and fatal states are saved?

Reply:
No, we should probably keep the same file paths. I'm think JRS might be hard coded to them. I didn't realize these strings were also generating the folder structure, thought it was just a logging thing.